javascript-loops/main.js

```diff
@@ -0,0 +1,83 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  for (var i = 0; i < 10; i++) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+console.log('getNumbers4,ToTen', getNumbersToTen());
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  for (var i = 2; i <= 20; i += 2) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+console.log('getEvenNumbersToTwenty:', getEvenNumbersToTwenty());
+
+function repeatWord(word, times) {
+  var count = 1;
```

Comment:
This variable is unnecessary, just set `i` to `1` in the initialization of your loop